kube/services/revproxy/gen3.nginx.conf/arborist-service.conf

```diff
@@ -27,6 +27,34 @@ location = /gen3-authz {
     client_max_body_size 0;
 }
 
+#
+# authorization endpoint
+# https://hostname/authz?resource=programs/blah&method=acb&service=xyz
+#
+location ~ /authz/? {
+    error_page 400 =403 @errorworkspace;
+    error_page 500 =403 @errorworkspace;
+
+    # avoid setting $upstream in authz subrequests ...
+    set $upstream_authz http://${arborist_release_name}-service.$namespace.svc.cluster.local;
```

Comment:
change this variable to just `$upstream` - that goes into the logs - the upstream_authz is particular to the authz flow thing